salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc

```diff
@@ -42,9 +42,18 @@ You must know the scope of the rate limiting policy that applies to your account
 
 When extracting data from a Salesforce Opportunity object, be aware that a "quarter" in this context is relative to the financial year of the organization and not necessarily to the calendar year.
 
+== Japanese Support
+
+Salesforce Connector supports Japanese characters for most operations, with these exceptions:
+
+* The *Publish Streaming Channel* operation does not support Japanese characters in the *Streaming Channel Name*, which can contain alphanumeric and `&#95;`, `/` characters only. See https://developer.salesforce.com/docs/atlas.en-us.228.0.object_reference.meta/object_reference/sforce_api_objects_streamingchannel.htm[Salesforce Streaming channel documentation].
```

Comment:
*Streaming Channel Name* field

This is not clear: "...which can contain alphanumeric and _, / characters only" Do you mean to say this? "...which contains alphanumeric characters and the `_ ` or `/` characters only"?